net-misc/pingu/files/pingu.initd

```diff
@@ -0,0 +1,10 @@
+#!/sbin/openrc-run
+
+description="Policy routing daemon"
+command=/usr/sbin/pingu
+command_args="-d ${OPTS}"
+pidfile=/var/run/pingu/pingu.pid
```

Comment:
/var is a link to /run.

Just use /run.

Reply:
Done.